### PR TITLE
Flavor text encoding fix

### PR DESCRIPTION
**Pull request by Markolie**
Changes:
1) Fixes encoding issues in flavour texts (i.e. ' not showing up properly).

### CODE REVIEW

File: code/modules/client/preferences.dm [hunk 1]
```diff
@@ -337,7 +337,7 @@ datum/preferences
 					else
 						dat += "[flavor_text]"
 				else
-					dat += "[copytext(flavor_text, 1, 37)]...<br>"
+					dat += "[TextPreview(flavor_text, 1, 37)]...<br>"
```
Comment (tigercat2000) [inline code/modules/client/preferences.dm]: Uh, this won't work. TextPreview only has two arguments, and this is setting len to 1.

**Comment by Spacemanspark** [comment]
:+1:

**Comment by Fox-McCloud** [comment]
Oh, you mean the

"a 5^&8@#4 red fox" thing?

**Comment by Markolie** [comment]
Yep.

**Comment by Fox-McCloud** [comment]
Hallelujah--thanks.

**Comment by KasparoVy** [comment]
Lmfao thanks a ton

**Pull request merged**